apps/account/templates/a4_candy_account/account_dashboard.html

```diff
@@ -43,6 +43,13 @@
                             {% trans 'Account Connections' %}
                         </a>
                     </li>
+                    <li class="dashboard-nav__page">
+                        <a href="{% url 'user_agreements' %}"
+                           class="dashboard-nav__item dashboard-nav__item--interactive {% if request.resolver_match.url_name == "user_agreements" %}is-active{% endif %}">
+                            <i class="far fa-square-check" aria-hidden="true"></i>
```

Comment:
@khamui And a question for you: what is the deal with the a+ icons? We don't use font awesome? (I saw an issue about cleaning them up...) How do we get them?

Reply:
was that the issue to clean up our custom icons? if so it was to create a clean set of custom icons with only the ones we needed but we pretty much only use them in interactive event

Reply:
Yes, that issue is what made me think that we don't use font awesome. But then I only added it wrong and we do use font awesome?

Reply:
So, we do use font awesome! But somehow, I could not find the right name. :)

Reply:
yes we still use it, closest i could find is <i class="fas fa-check-square"></i> we haven't updated to 6 yet I don't think